app/src/interfaces/_system/system-modules/system-modules.vue

```diff
@@ -114,174 +113,159 @@ const linkFields: DeepPartial<Field>[] = [
+}>();
+
+const { t } = useI18n();
+
+const editing = ref<string | null>();
+const values = ref<SettingsModuleBarLink | null>();
+const initialValues = ref<SettingsModuleBarLink | null>();
+
+const { modules: registeredModules } = useExtensions();
+
+const availableModulesAsBarModule = computed<SettingsModuleBarModule[]>(() => {
+	return registeredModules.value
+		.filter((module) => module.hidden !== true)
+		.map(
+			(module): SettingsModuleBarModule => ({
+				type: 'module',
+				id: module.id,
+				enabled: false,
+			})
+		);
+});
 
-		const valuesWithData = computed<PreviewValue[]>({
-			get() {
-				const savedModules = (
-					(props.value ?? MODULE_BAR_DEFAULT).filter((value) => value.type === 'module') as SettingsModuleBarModule[]
-				).map((value) => value.id);
-
-				return valueToPreview([
-					...(props.value ?? MODULE_BAR_DEFAULT),
-					...availableModulesAsBarModule.value.filter(
-						(availableModuleAsBarModule) => savedModules.includes(availableModuleAsBarModule.id) === false
-					),
-				]);
-			},
-			set(previewValue: PreviewValue[]) {
-				emit('input', previewToValue(previewValue));
-			},
-		});
+const valuesWithData = computed<PreviewValue[]>({
+	get() {
+		const savedModules = (
+			(props.value ?? MODULE_BAR_DEFAULT).filter((value) => value.type === 'module') as SettingsModuleBarModule[]
+		).map((value) => value.id);
+
+		return valueToPreview([
+			...(props.value ?? MODULE_BAR_DEFAULT),
+			...availableModulesAsBarModule.value.filter(
+				(availableModuleAsBarModule) => savedModules.includes(availableModuleAsBarModule.id) === false
+			),
+		]);
+	},
+	set(previewValue: PreviewValue[]) {
+		emit('input', previewToValue(previewValue));
+	},
+});
 
-		const isSaveDisabled = computed(() => {
-			for (const field of linkFields) {
-				if (field.meta?.required && field.field) {
-					const fieldValue = (values.value as Record<string, any>)[field.field];
-					if (fieldValue === null || fieldValue === undefined || fieldValue === '') return true;
-				}
-			}
+const isSaveDisabled = computed(() => {
+	for (const field of linkFields) {
+		if (field.meta?.required && field.field) {
+			const fieldValue = (values.value as Record<string, any>)[field.field];
+			if (fieldValue === null || fieldValue === undefined || fieldValue === '') return true;
+		}
+	}
 
-			return false;
-		});
+	return false;
+});
 
-		return {
-			t,
-			editing,
-			valuesWithData,
-			hideDragImage,
-			updateItem,
-			edit,
-			linkFields,
-			isSaveDisabled,
-			save,
-			values,
-			remove,
-			initialValues,
-		};
+function valueToPreview(value: Settings['module_bar']): PreviewValue[] {
+	return value
+		.filter((part) => {
+			if (part.type === 'link') return true;
+			return !!registeredModules.value.find((module) => module.id === part.id);
+		})
+		.map((part) => {
+			if (part.type === 'link') {
+				return {
+					...part,
+					to: part.url,
+					icon: part.icon,
+					name: part.name,
```

Comment:
The `translate` function got applied to the wrong place!
Should be on this line instead of line 199.